### PR TITLE
Update monitors docs with Granular Access control

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do? What is the motivation?

- Add information on Granular Access control to Getting Started w/ Monitors
- Add permissions information for Monitors Configure page
- Move the existing permissions information from the Notification docs to the Configure docs page
- Redirect all links for the notification permissions to the Configure permissions page
- Add checkmark under team and user access for the [Granular Access Control ](https://docs.datadoghq.com/account_management/rbac/granular_access/#manage-access-to-individual-resources)table for Monitors
- [DOCS-8130](https://datadoghq.atlassian.net/browse/DOCS-8130)

### Merge instructions

**Do not merge**: Pending GA July 1

### Additional notes

[DOCS-8130]: https://datadoghq.atlassian.net/browse/DOCS-8130?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/account_management/rbac/granular_access.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ A user with the `user_access_manage` permission can elevate their access to any
 
 [1]: /account_management/teams/
 [2]: /dashboards/configure/#permissions
-[3]: /monitors/notify/#permissions
+[3]: /monitors/configure/#permissions
```
Comment (jhgilbert) [inline content/en/account_management/rbac/granular_access.md:31]:
```suggestion
[3]: /monitors/configuration/#permissions
```

**Review by jhgilbert — APPROVED** [review]
Approved with some suggestions, let me know if you need another look at any point. Thanks!

File: content/en/account_management/rbac/granular_access.md (hunk 1 above)
Comment (estherk15) [reply content/en/account_management/rbac/granular_access.md:31]: Good catch!

**Review by khangtruong — APPROVED** [review]
Looks good to me 👍

**Pull request merged**